sklearn/tests/test_common.py

```diff
@@ -265,7 +265,6 @@ def test_search_cv(estimator, check, request):
     'model_selection',
     'multiclass',
     'multioutput',
-    'pipeline',
```

Comment:
remark: even if removed from the list, the estimators of this module are not tested anyway (skipped in the instance constructions)

Reply:
It is true that we only have a negative test for `n_features_in_` via `sklearn.tests.test_metaestimator.test_meta_estimators_delegate_data_validation` and `_generate_meta_estimator_instances_with_pipeline` that is updated in this pipeline.

It would be great to have a new positive common test for the presence of `n_features_in_` on tabular dataset in `sklearn.tests.test_metaestimator`.